javascript/chronometer.js

```diff
@@ -1,10 +1,68 @@
 class Chronometer {
   constructor() {
+    
     // ... your code goes here
+  
+   
+        this.currentTime = 0;
+        this.intervalId = null ;
+      }
+    
+      start(callback) {
+        this.intervalId=setInterval(()=> {
+         callback()
+         this.currentTime ++ },1000)
+      }
+      getMinutes() {
+        return Math.floor(this.currentTime/60);
+      }
+    
+      getSeconds() {
+         return (this.currentTime)%60; // this modulo will take the minutes out and show the remainder in seconds
+      }
+    
+      computeTwoDigitNumber(value) {
+        if(value < 10) return `0${value}`;
+        else return `${value}`;
+      }
+    
+      stop() {
+        clearInterval(this.intervalId);
+      }
+    
+      reset() {
+        this.currentTime = 0;
+      }
+    
+      split() {
+        
+        return `${this.computeTwoDigitNumber(this.getMinutes())}:${this.computeTwoDigitNumber(this.getSeconds())}`;
+      }
+    }
+
+  
+  // The following is required to make unit tests work.
+  /* Environment setup. Do not modify the below code. */
+  if (typeof module !== 'undefined') {
+    module.exports = Chronometer;
   }
+    
```

Comment:
Hi Sina! You need to delete everything after line 49 in chronometer.js.
You have every function twice. The tests for chronometer.js will not work if you have the functions etc. more than one time.
Again: tests are only for chronometer.js, we don't have tests for index.js